### ISSUE TITLE
Code related discussion of PnP going on on Yammer

### ISSUE THREAD

**Issue by pbjorklund** [SharePoint/PnP#83]
It is my humble belief that all actual code and project related discussions should be hosted on github.

I have no issue with having Yammer being the place for getting help with things that are not bugs, announcing new publicly available builds or posting interesting related information. I.e, make Yammer the "consumer" channel. 

I think it would be nice to use github as the "developer" channel. For instance things like the below should be filed as an (or two) issues here on github:

"PnP contain samples that upload to and reference files in the 15 path. But I also see references to 16 and I see people with broken SPO customizations. In order to build a future proof app any paths used must be stable. Two questions:

How future proof is the 15 folder?

What is the best practice in choosing where to upload resource files such as css and images that should be globally accessed by sites and subsites?"

To sum up: It would be nice to have consumer (yammer) and developer (github) channels in the project.

What do you other guys and girls think?
